docs/index.md

```diff
@@ -0,0 +1,45 @@
+## Introduction
+
+Welcome to the `vscode-debug-adapter-apache-camel` project created by Red Hat! Here you'll find a description of how to use VS Code extension providing Debug Adapter for Apache Camel.
+
+## Description
+
+This extension adds <a href="https://camel.apache.org/manual/debugger.html">Camel Debugger</a> power by attaching to a running Camel route written in Java, Yaml or XML DSL directly in your Visual Studio Code editor. It is working as a client using the [Microsoft Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol/) which communicates with [Camel Debug Server](https://github.com/camel-tooling/camel-debug-adapter) providing all functionalities.
+
+## How to install
+
+1. You can download this **vscode-debug-adapter-apache-camel** extension from the [VS Code Extension Marketplace](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-debug-adapter-apache-camel) and the [Open VSX Registry](https://open-vsx.org/extension/redhat/vscode-debug-adapter-apache-camel).
+2. Debug Adapter for Apache Camel can be also installed directly in the [Microsoft VS Code](https://code.visualstudio.com/).
+
+    **Steps**
+    - Open your VS Code.
+    - In VS Code, select **View > Extensions**.
+    - In the search bar, type `Camel`
```

Comment:
Extension id would be more specific: `redhat.vscode-debug-adapter-apache-camel`

Reply:
I dont like that I need to write (copy&paste) whole precise id otherwise it is not found

I changed to `Camel Debug` to ensure our extension is listed as first search resultotherwise